core/src/test/java/org/elasticsearch/get/GetActionIT.java

```diff
@@ -585,7 +587,7 @@ public void testGetFieldsNonLeafField() throws Exception {
                 .get();
 
         try {
-            client().prepareGet(indexOrAlias(), "my-type1", "1").setFields("field1").get();
+            client().prepareGet(indexOrAlias(), "my-type1", "1").setStoredFields("field1").get();
             fail();
         } catch (IllegalArgumentException e) {
             //all well
```

Comment:
Can you add an assertion here? Maybe rewrite to `expectThrows` if you are feeling brave. But I really would like an assertion about the message here regardless.